src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilter.java

```diff
@@ -170,6 +173,21 @@ private static GATKRead clearReadAlignment(final GATKRead read, final SAMFileHea
                 });
     }
 
+    /**
+     * Validate arguments
+     */
+    private void validateFilterArguments() {
+        final SAMSequenceDictionary dictionary = header.getSequenceDictionary();
+        if (filterArgs.alignedInput) {
+            final Set<String> contigsToIgnoreSet = new HashSet<>(filterArgs.alignmentContigsToIgnore);
+            for (final String contig : contigsToIgnoreSet) {
+                if (dictionary.getSequence(contig) == null) {
+                    throw new UserException.MissingContigInSequenceDictionary(contig, dictionary);
```

Comment:
I'd like for the error to specify somehow is that the problem is the contigs specified for filtering.  Otherwise users will be validating all their input files and scratching their heads.

Reply:
Done